app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -120,13 +120,17 @@ def textual_group_endpoints
     )
   end
 
-  def textual_group_miq_custom_attributes
-    TextualGroup.new(_("Custom Attributes"), textual_miq_custom_attributes)
+  def textual_group_miq_displayable_custom_attributes
+    TextualGroup.new(_("Custom Attributes"), textual_miq_displayable_custom_attributes)
   end
 
-  def textual_miq_custom_attributes
-    attrs = @record.custom_attributes
+  def textual_miq_displayable_custom_attributes
+    attrs = @record.custom_attributes.where(
```

Comment:
It would be good to keep this consistent with other textual_summaries, so we can then extract this code to common mixin.

So in other textual_summaries we use miq_custom_attributes:
https://github.com/Ladas/manageiq/blob/b18c5104f6dee66a2406687919ba5e5f4c84e7ec/app/models/mixins/custom_attribute_mixin.rb#L8-L8

So best if you could use different :source for attributes that should not be displayable and pack it to a model relation?